blog/arquillian-pact-1.0.0.Alpha2.textile

```diff
@@ -0,0 +1,73 @@
+---
+date: 2016-09-29
+author: lordofthejars
+---
+
+p. Arquillian Pact is an extension that provides an integration between Arquillian and "Pact":https://docs.pact.io/.
+
+h3 What's new
+
+p. <i class="icon-star"></i> Maven Loader
+
+p. Now you can use any Maven artifact to store contracts (pacts files). Then provider will resolve given artifact, uncompress it and execute the contracts found there.
```

Comment:
> uncompress it and execute the contracts found there.

This is implementation detail - we can just say that provider will resolve the artifact and use the files included there.